compiler/rustc_const_eval/src/transform/check_consts/resolver.rs

```diff
@@ -265,6 +273,17 @@ pub(super) struct State {
     pub borrow: BitSet<Local>,
 }
 
+impl Clone for State {
```

Comment:
Please put the comment from the commit on this impl, so future readers don't have to look at blame